Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
What's the thinking behind needing both static and shared? Does the `make install` stage not essentially overwrite a lot of what was installed in the previous compile aside the dynamic libraries? If you build with both static and dynamic, which set of libraries do the executables hook into? Both?